js/src/dht/findpeer.js

```diff
@@ -38,10 +38,9 @@ module.exports = (createCommon, options) => {
     after((done) => common.teardown(done))
 
     it('should find other peers', (done) => {
-      nodeA.dht.findpeer(nodeB.peerId.id, (err, peer) => {
+      nodeA.dht.findpeer(nodeB.peerId.id, (err, res) => {
         expect(err).to.not.exist()
-        // TODO upgrade the answer, format is weird
-        expect(peer[0].Responses[0].ID).to.be.equal(nodeB.peerId.id)
+        expect(res.Responses[0].Addrs).to.deep.include(nodeB.peerId.addresses[0])
```

Comment:
is `deep` necessary - the values are just strings right?

Reply:
Can we please fix the object that is returned here to not have go-style uppercased first letter properties? - for consistency with the rest of the API.

Reply:
According to http api [docs](https://docs.ipfs.io/reference/api/http/#api-v0-dht-findpeer), the `addrs` is an array of addresses. that's why I am using the `deep.include`.

Changed the uppercase letter. However, the inconsistencies will have to be solved in `ipfs-api` in [ipfs/js-ipfs-api#890](https://github.com/ipfs/js-ipfs-api/pull/890). `go-ipfs` sends the uppercase style, and `js-ipfs` (aiming to have its core implementation compliant with this interface) must send its response in lowercase. As a result, `ipfs-api` using `js-ipfs` will receive the lowercase ones. I added the logic to deal with this inconsistency in the `ipfs-api` PR, but I am not sure if it is the better approach in this case.